### PR TITLE
KOGITO-3053 Upgrade kogito-apps to Quarkus 1.7

### PR DESCRIPTION
**Pull request by cristianonicolai**
With this upgrade, we move to Infinispan client 11 but running againt version 10.
I created a followup story to handle just the Infinispan migration, see [KOGITO-3118](https://issues.redhat.com/browse/KOGITO-3118)

### CODE REVIEW

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by radtriste — APPROVED** [review]
change looks good. let's see the pipeline ...
@cristianonicolai some sonarcloud issues as well

**Comment by cristianonicolai** [comment]
@radtriste deprecation issue mentioned by Sonar will be handled by https://issues.redhat.com/browse/KOGITO-3119

File: pom.xml [hunk 1]
```diff
@@ -16,17 +16,14 @@
...
-    <version.io.quarkus>1.6.0.Final</version.io.quarkus>
     <version.frontend.plugin>1.9.1</version.frontend.plugin>
     <version.node>v12.16.2</version.node>
     <version.npm>6.10.3</version.npm>
     <version.yarn>v1.19.1</version.yarn>
     <maven.compiler.release>11</maven.compiler.release>
     <version.maven-remote-resources-plugin>1.7.0</version.maven-remote-resources-plugin>
 
-    <version.org.infinispan>10.1.5.Final</version.org.infinispan>
     <version.org.infinispan.image>10.1.5.Final</version.org.infinispan.image>
```
Comment (danielezonca) [inline pom.xml:26]: Let's update the images too
```suggestion
    <version.org.infinispan.image>${version.org.infinispan}</version.org.infinispan.image>
```
or 
```suggestion
    <version.org.infinispan.image>11.0.1.Final</version.org.infinispan.image>
```

Comment (cristianonicolai) [reply pom.xml:26]: @danielezonca that will break :) There is a story just to handle all that is needed to do that https://issues.redhat.com/browse/KOGITO-3118

**Review by Sgitario — APPROVED** [review]

**Review by danielezonca — APPROVED** [review]

**Review by tiagodolphine — APPROVED** [review]

**Pull request merged**